### ISSUE TITLE
Alt text for blog cover image

### ISSUE THREAD

**Issue by vilbergs** [vilbergs/blog-v9001#2]
All images should have relevant alt text all throughout the page.

**Suggested Solution:** provide an alt text image via blog post frontmatter
